provisioner/remote_files_test.go

```diff
@@ -172,7 +172,7 @@ func TestMakeArchive(t *testing.T) {
 			Encrypted:   true,
 		},
 	} {
-		t.Run(tc.Message, func(tt *testing.T) {
+		t.Run(tc.Message, func(_ *testing.T) {
```

Comment:
This should be `t` for `require.NoError(t, err)` below

Reply:
Good catch. Thanks!